sdk/ai/azure-ai-resources/azure/ai/resources/client/_ai_client.py

```diff
@@ -315,16 +315,16 @@ def build_index_on_cloud(
             IndexSource,
             IndexStore,
         )
-        from azure.ai.resources._index._embeddings.EmbeddingsContainer import from_uri
+        from azure.ai.resources._index._embeddings import EmbeddingsContainer
         if isinstance(input_source, ACSSource):
             from azure.ai.resources._index._utils.connections import get_connection_by_id_v2, get_target_from_connection
 
             # Construct MLIndex object
             mlindex_config = {}
             connection_args = {"connection_type": "workspace_connection", "connection": {"id": aoai_connection_id}}
-            mlindex_config["embeddings"] = from_uri(
+            mlindex_config["embeddings"] = EmbeddingsContainer.from_uri(  # type: ignore[attr-defined]
                 build_open_ai_protocol(embeddings_model), **connection_args
-            ).get_metadata()
+            ).get_metadata()  # Bug 2922096
```

Comment:
FYI: added a bug fix work item here discussed offline with @jingyizhu99 who is working on fixing bugs in these files. The bug fix is out of scope for this PR.